.github/workflows/build.yml

```
@@ -37,3 +37,28 @@ jobs:
         with:
           reports: |
             reference/**/build/reports/checkstyle/*.xml
+
+# :::automatic-testing:game-test:3
+  client_game_test:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'microsoft'
+          java-version: '21'
+
+      - name: Run production client gametest
```

Let's try running the action again. Is 10 minutes a good max time?

```suggestion
      - name: Run production client gametest
        timeout-minutes: 10
```